lib/transport/transport.ts

```diff
@@ -3,11 +3,21 @@
 
 import * as transportable from './transportable';
 import * as functionTransporter from './function-transporter';
+import * as v8TransportHelper from './v8-transport-helper';
 import * as path from 'path';
 
 /// <summary> Per-isolate cid => constructor registry. </summary>
 let _registry: Map<string, new(...args: any[]) => transportable.Transportable> 
-    = new Map<string, new(...args: any[]) => transportable.Transportable>(); 
+    = new Map<string, new(...args: any[]) => transportable.Transportable>();
+
+let _v8TransportHelperVerifiedWhitelist = new Set();
```

Comment:
_builtInTypeWhiteList?

Reply:
_verifiedBuiltInTypeWhitelist ?

---
In reply to: [154740102](https://github.com/Microsoft/napajs/pull/137#discussion_r154740102) [](ancestors = 154740102)

Reply:
let's use builtInTypeWhiteList

---
In reply to: [155847067](https://github.com/Microsoft/napajs/pull/137#discussion_r155847067) [](ancestors = 155847067,154740102)